cmd/ingress.go

```diff
@@ -35,7 +35,7 @@ func (a *MethodK8s) InitIngressCommand() {
 			a.OutputSignal.Content = report
 		},
 	}
-	enumerateCmd.Flags().Bool("gateways", false, "Only include gateway objects")
+	enumerateCmd.Flags().String("objects", "ingress,gateway", "Provided a comma seperated list of the objects you want enumerated (ie.ingress). The default is 'ingress,gateway'")
```

Comment:
I think you should try to use this as a string slice (array) and not as a comma delimited string. Webscan has an example. https://github.com/Method-Security/webscan/blob/develop/cmd/vuln.go#L40

I'm _pretty_ sure this lets you do something like `--objects gateway --objects ingress` which is a lot cleaner UX. Also, let's maybe use something other than `objects` as that's not really clear. First thought was `targets` but we use that for other things in other CLIs. Perhaps, `types`?